src/Html.php

```diff
@@ -1956,9 +1957,16 @@ public static function getNonArrayableName(string $name): string
      * Render attribute in HTML tag.
      *
      * @link https://html.spec.whatwg.org/#a-quick-introduction-to-html
+     * @throws ValueError When name contains any of the following: SPACE " ' > / =
      */
     private static function renderAttribute(string $name, string $encodedValue = '', string $quote = '"'): string
     {
+        if (preg_match('/[\s"\'>\/\=]/', $name) > 0) {
```

Comment:
Better to use `str_contains` instead of regular expression for this case.

Reply:
@vjik nope.

Reply:
Do you also want checks for [https://infra.spec.whatwg.org/#noncharacter](url) and [https://infra.spec.whatwg.org/#control](url) ?

Reply:
That would be good.